test/fixtures/e2e/amp-list/amp-list.ssr.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html ⚡ allow-viewer-render-template>
+<head>
+    <meta charset="utf-8">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+    <style amp4email-boilerplate>body{visibility:hidden}</style>
+</head>
+<body>
+
+  <amp-state id="data">
+    <script type="application/json">
+    {
+      "fruit": "Pineapple"
+    }
+    </script>
+  </amp-state>
```

Comment:
What's this `amp-state` element for?

Reply:
amp-state is used to test bindings for the ssr test case. It is not applied in the non ssr use case.